src/resources/entities/EnemyManager.py

```diff
@@ -32,8 +34,3 @@ def update(self, x: int, y: int) -> None:
     def collisions_with_player(self, x: int, y: int) -> bool:
         """Checks if player collided with enemy"""
         return any([(enemy.x, enemy.y) == (x, y) for enemy in self.enemy_list])
```

Comment:
Can this be also refactored to Entity. The Color Changer class would use this, since the player is the only entity that needs to be tracked by Color Changer

Reply:
Collisions might be refactored to the god class that is Game Resources, lol. Something should probably be done about that class :/